api/src/main/java/org/pmiops/workbench/api/SumoLogicController.java

```diff
@@ -0,0 +1,113 @@
+package org.pmiops.workbench.api;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.logging.Logger;
+import javax.inject.Provider;
+import org.pmiops.workbench.actionaudit.auditors.SumoLogicAuditor;
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.pmiops.workbench.exceptions.BadRequestException;
+import org.pmiops.workbench.exceptions.UnauthorizedException;
+import org.pmiops.workbench.google.CloudStorageService;
+import org.pmiops.workbench.model.EgressEvent;
+import org.pmiops.workbench.model.EgressEventRequest;
+import org.pmiops.workbench.model.EmptyResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class SumoLogicController implements SumoLogicApiDelegate {
+
+  public static final String SUMOLOGIC_KEY_FILENAME = "inbound-sumologic-keys.txt";
```

Comment:
I'd make this a per-env property most likely. Can we simulate the events on Test and Local?